frontend/test/model/common/ValidationErrorIcon.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { ThemeProvider } from '@mui/material/styles'
+import { render, screen, waitFor } from '@testing-library/react'
+import { MarkdownDisplayProps } from 'src/common/MarkdownDisplay'
+import ValidationErrorIcon from 'src/model/beta/common/ValidationErrorIcon'
+import { lightTheme } from 'src/theme'
+import { testAccessRequestSchemaStepNoRender } from 'utils/test/testModels'
+import { describe, expect, it, vi } from 'vitest'
+
+vi.mock('src/common/MarkdownDisplay.tsx', () => ({ default: (_props: MarkdownDisplayProps) => <></> }))
+
+describe('ValidationErrorIcon', () => {
+  it('shows a validation warning message and icon when the form step is marked as incomplete', async () => {
```

Comment:
You've said it should show a message and icon, but are only asserting that the message is shown. You should add another assertion to check for the icon.